static/js/common/upload-addon.js

```diff
@@ -431,6 +431,20 @@
                       $("<p>").text(gettext("Your submission will be automatically signed.")).appendTo(upload_results);
                     }
 
+                    if (results.validation.is_upgrade_to_webextension) {
+                        var warning_box = $('<div>').attr('class', 'important-warning').appendTo(upload_results);
```

Comment:
Oops, didn't see that comment. I'll make a note to clean that up once we touch that file again.